src/app/GitUI/CommandsDialogs/Menus/ToolsToolStripMenuItem.cs

```diff
@@ -39,6 +39,8 @@ public override void RefreshShortcutKeys(IEnumerable<HotkeyCommand>? hotkeys)
         public override void RefreshState(bool bareRepository)
         {
             gitGUIToolStripMenuItem.Enabled = !bareRepository;
+            kGitToolStripMenuItem.Enabled = !bareRepository;
```

Comment:
GitK works well for bare repos, and it doesn't crash if there is no repo.

![image](https://github.com/user-attachments/assets/657e6da8-94fa-412f-9a5a-103a7ed8b450)

Would it really be worth passing this information to `RefreshState(bool bareRepository)`?